apiary.apib

```diff
@@ -1884,17 +1884,6 @@ Retrieves information related to the current organization account.
             "StatusID": "ACTIVE",
             "Contact": null,
             "Email": null,
-            "PasswordMinLength": 5,
-            "AdminPasswordMinLength": 5,
-            "PasswordRequireCharacterClasses": 0,
-            "PasswordRequireUppercase": false,
-            "PasswordRequireLowercase": false,
-            "PasswordRequireDigits": false,
-            "PasswordRequireSpecial": false,
-            "PasswordRotateDays": 0,
-            "RequireUniquePasswordsCount": 0,
-            "SessionMaxDurationSeconds": 0,
-            "SessionIdleDurationSeconds": 0,
             "OrgAddresses": [],
             "LivenessConfig": {
```

Comment:
The response is missing the following param:
- Logo
- BrandID
- ConsentId

"LivenessConfig" is not part of the response, please remove.